CICD.Tools.CatalogUpload.Lib/CatalogService/CatalogYaml.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace Skyline.DataMiner.CICD.Tools.CatalogUpload.Lib.CatalogService
+{
+	using System.Collections.Generic;
+
+	/// <summary>
+	/// Represents the metadata of a catalog entry in YAML format.
+	/// </summary>
+	public class CatalogYaml
+	{
+		/// <summary>
+		/// Initializes a new instance of the <see cref="CatalogYaml"/> class.
+		/// </summary>
+		public CatalogYaml()
+		{
+			Owners = new List<CatalogYamlOwner>();
+			Tags = new List<string>();
+		}
+
+		/// <summary>
+		/// Gets or sets the URL to the documentation related to the catalog entry.
+		/// </summary>
+		/// <value>A string representing the documentation URL.</value>
+		public string Documentation_url { get; set; }
```

Comment:
Make use of the YamlMember attribute [YamlMember(Alias = "Documentation_url")]

Reply:
Using 		var deserializer = new DeserializerBuilder()
		.WithNamingConvention(**UnderscoredNamingConvention**.Instance)  // see height_in_inches in sample yml
		.Build();

Reply:
done